src/Makefile

```diff
@@ -21,7 +21,11 @@ timg : $(OBJECTS)
 timg.o : timg-version.h
```

Comment:
Wait, wouldn't this Makefile be removed ? Maybe this snug back in while rebasing ?
Also, the Makefile in `man/` can go as well now as we now generate it with cmake.

Reply:
Yepp.. Likely. However, they are gone now.